pkg/config/types/duration.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 type Duration struct {
-	time.Duration
+	time.Duration `json:",inline"`
```

Comment:
Can we move or recreate this time in `api/common` next to the `targetref.go`?

Reply:
no longe apply since I don't use this duration anymore